UnitySetup/UnitySetup.psm1

```diff
@@ -1870,7 +1870,36 @@ function Start-UnityEditor {
             $process.Start() | Out-Null
 
             if ( $Wait ) {
-                $process.WaitForExit()
+                if( $LogFile ) {
+                    $ljob = Start-Job -ScriptBlock {
+                        param($log)
+
+                        while ( -not (Test-Path $log -Type Leaf) ) {
+                            Start-Sleep -Milliseconds 100
+                        }
+
+                        Get-Content "$log" -Wait
+                    } -ArgumentList $LogFile
+
+                    while ( -not $process.HasExited )
+                    {
+                        do {
+                            $logOutput = Receive-Job $ljob
+                            $logOutput | Write-Host
+                        } until($logOutput.Length == 0)
```

Comment:
@jwittner this [line is invalid for me.](https://dev.azure.com/xrtk/Mixed%20Reality%20Toolkit/_build/results?buildId=3602&view=logs&j=0ab14b9f-e499-56d5-97b1-fd98b70ea339&t=f064c65f-5d7b-5dd9-a2c0-b27c2b3dbefa&l=24) Is there another way to write this?

Reply:
@jwittner any ideas?

Reply:
Sorry yeah, was pseudocode. =)

Try `} until ($logOutput.Length -eq 0)`